prov/sm2/src/sm2_attr.c

```diff
@@ -126,8 +127,8 @@ struct fi_domain_attr sm2_hmem_domain_attr = {
 struct fi_fabric_attr sm2_fabric_attr = {.name = "sm2",
 					 .prov_version = OFI_VERSION_DEF_PROV};
 
-struct fi_info sm2_hmem_info = {.caps =
-				    SM2_HMEM_TX_CAPS | SM2_HMEM_RX_CAPS | FI_MULTI_RECV,
+struct fi_info sm2_hmem_info = {.caps = SM2_HMEM_TX_CAPS | SM2_HMEM_RX_CAPS |
```

Comment:
Clang did not play well with this....

Reply:
Yea, I noticed that too... may need to investigate setting to figure out how to deal with structures better.

Reply:
We fixed clang-format file, and added , to the end of the structures